_posts/2023-08-23-cluster-metrics-monitors-blog.md

```diff
@@ -0,0 +1,168 @@
+---
+layout: post
+title:  Monitoring your cluster with cluster metrics monitors
+authors:
+  - hurneyt
+date: 2023-08-23
+categories:
+  - technical-posts
+meta_keywords: 
+meta_description: 
+---
```

Comment:
Please include the following meta: 

**meta_keywords:** cluster metrics monitor, monitor OpenSearch cluster metrics, alerting plugin for OpenSearch
**meta_description:** Learn about common use cases for cluster metrics monitors in OpenSearch how you can monitor CPU utilization and JVM memory pressure with _cat/indices and _cat/shards APIs

Reply:
Once the meta is added and the blog date is updated, we can publish.

Reply:
@pajuric Done